mino/minows/address.go

```diff
@@ -86,7 +86,7 @@ func (f addressFactory) FromText(text []byte) mino.Address {
 	}
 	addr, err := newAddress(location, identity)
 	if err != nil {
-		dela.Logger.Err(xerrors.Errorf("could not create address: %v", err))
+		dela.Logger.Err(xerrors.Errorf("could not create address: %w", err))
```

Comment:
Please revert. The standard error-passing mechanism in dela is using `%v`.

Reply:
changed, though i'm not sure how this makes error assertion still possible.

Reply:
Not sure, either. But sometimes it's more important to use common style than to support a feature nobody uses.

If you think that `errors.Is()` is not possible, open an issue on the dedis/dela repository, and express your concerns.